src/styles.scss

```diff
@@ -208,4 +280,6 @@ h4 {
 
 /* Questions ---------------------(start) */
 
-/* Questions ---------------------(end) */
\ No newline at end of file
+/* Questions ---------------------(end) */
+
+/* Questions ---------------------(end) */
```

Comment:
Why does this appear twice?

Reply:
I left as it was just in case because I didn't create this one. 
But it seems unnecessary, so, I will delete this one now!